src/main/java/com/openlattice/chronicle/controllers/ChronicleStudyController.java

```diff
@@ -130,15 +131,51 @@ public Boolean isKnownParticipant(
         return chronicleService.isKnownParticipant( studyId, participantId );
     }
 
+    @RequestMapping(
+            path = STUDY_ID_PATH + PARTICIPANT_ID_PATH,
+            method = RequestMethod.DELETE
+    )
     @Override
-    public void deleteParticipantAndAllNeighbors(
-            UUID studyId, String participantId, DeleteType deleteType ) {
-        // TODO implement this
+    public Void deleteParticipantAndAllNeighbors(
+            @PathVariable( STUDY_ID ) UUID studyId,
+            @PathVariable( PARTICIPANT_ID ) String participantId,
+            @RequestParam( TYPE ) DeleteType deleteType
+    ) {
+        if ( deleteType == null )
+            deleteType = DeleteType.Soft;
+
+        Authentication authentication = SecurityContextHolder.getContext().getAuthentication();
+        if ( authentication instanceof JwtAuthentication ) {
```

Comment:
Instead of checking whether there's valid jwt authentication here, I'd add a line to `ChronicleSecurityPod.authorizeRequests`, i.e.
```
.antMatchers( "/chronicle/your/path/**" ).authenticated()
```
and then any requests that aren't authenticated will throw an exception before entering the controller method. It may even be worth adding an `/authenticated` path to any requests that need a JWT, just to be explicit about requirements on all sides, and so that you can use the path "/chronicle/authenticated/**"

Reply:
https://github.com/openlattice/chronicle-api/pull/30
https://github.com/openlattice/chronicle-server/pull/102